### DISCUSSION TITLE
Windows 10 "Similar Images" comparing both JPG and HEIC file types

### DISCUSSION THREAD

**Discussion by KnightCa** [qarmin/czkawka#987] [Q&A]
Cskawka version: 5.1.0
OS: Windows 10 64-bit

I have seen that the latest version supports HEIC files. When I run a search for "Similar Images", it does not seem to include HEIC files in it's finding duplicate images. How do I get this to work on a Windows 10 OS?

**Comment by qarmin** [comment]
I tried to compile app with support of checking this files in Windows, but looks that app fails to compile due missing libheif library. 

https://github.com/qarmin/czkawka/actions/runs/5229139294/jobs/9441950086

```
error: could not compile `czkawka_cli` (bin "czkawka_cli") due to previous error
warning: build failed, waiting for other jobs to finish...
error: linking with `x86_64-w64-mingw32-gcc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/github/home/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/github/home/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" VSLANG="1033" "x86_64-w64-mingw32-gcc" "-fno-use-linker-plugin" "-Wl,--dynamicbase" "-Wl,--disable-auto-image-base" "-m64" "-Wl,--high-entropy-va" "/github/home/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib
...
"-luser32" "-lkernel32" "-Wl,--nxcompat" "-L" "/github/home/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-pc-windows-gnu/lib" "-o" "/__w/czkawka/czkawka/target/x86_64-pc-windows-gnu/release/deps/czkawka_gui-838c44d70518f4eb.exe" "-Wl,--subsystem" "-Wl,windows" "-Wl,--gc-sections" "-no-pie" "-Wl,-O1" "-nodefaultlibs" "/github/home/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-pc-windows-gnu/lib/rsend.o"
  = note: /usr/lib/gcc/x86_64-w64-mingw32/12.2.1/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lheif: No such file or directory
          collect2: error: ld returned 1 exit status
          

warning: `czkawka_gui` (bin "czkawka_gui") generated 1 warning
error: could not compile `czkawka_gui` (bin "czkawka_gui") due to previous error; 1 warning emitted
Error: Process completed with exit code 101.
```

Probably here should be added package `mingw-libheif` which unfortunately not exists in fedora yet (https://packages.fedoraproject.org/index/mi.html - I cannot find `libheif`) - this package exists e.g. in arch AUR - https://aur.archlinux.org/packages/mingw-w64-libheif
https://github.com/qarmin/czkawka/blob/55b2744bf4d97d6fafabbfd6c64d103551ede056/.github/workflows/windows.yml#L20

**Comment by pronak657** [comment]
@KnightCa Were you able to get it to work? 

I downloaded latest 6.0.0 version and I'm facing similar issue

**Comment by KnightCa** [comment]
I was not able to get it to work. Ended up translating the HEIC images to jpgs, then did the comparison tool with them as the “master”, then deleted the jpg conversions after. Not ideal.On Jul 7, 2023, at 8:10 PM, Ronak Patel ***@***.***> wrote:﻿
@KnightCa Were you able to get it to work?
I downloaded latest 6.0.0 version and I'm facing similar issue
